submit/samples/sample_checklist/sample_checklist_errors.md

```diff
@@ -0,0 +1,24 @@
+|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Error at {<sample-name>}. Invalid units "{value-of-units}" for field "{<attribute-name>}". Allowed units are "{<list-of-allowed-values>}"                                        | sample:{<sample-name>}; attribute:{<path-to-attribute>}; reason:must be equal to one of the allowed values:{list-of-allowed-values}                                                        |
+| Missing units for field "{<attribute-name>}". Allowed units are "{<list-of-allowed-values>}"                                                                                     | sample:{<sample-name>}; attribute:{<path-to-attribute>}; reason:must have required property 'unit'                                                                                         |
+| Error at {<sample-name>}. The length of field "{<attribute-name>}" value "{<value-provided>}" exceeds the maximum allowed length "{<maximum-length-of-field>}".                  | sample:{<sample-name>}; attribute:{<path-to-attribute>}; reason:must NOT have more than {<maximum-length-of-field>} characters                                                             |
+| Error at {<sample-name>}. The value "{<value-provided>}" for field "{<attribute-name>}" does not match regular expression "{<pattern>}"                                          | sample:{<sample-name>}; attribute:{<path-to-attribute>}; reason:must match pattern {<pattern>}                                                                                             |
+| Missing mandatory field "{<property-name>}".                                                                                                                                     | sample:{<sample-name>}; attribute:{<path-to-attribute>}; reason:must have required property {<property-name>}                                                                              |
+| Missing recommended field "{<property-name>}".                                                                                                                                   | **DEPRECATED**. Recommended field validation will no longer be supported.                                                                                                                  |
+| Error at {<sample-name>}. Field "{<property-name>}" may only occur once.                                                                                                         | sample:{<sample-name>}; attribute:{<path-to-attribute>}; reason:must NOT have more than 1 items                                                                                            |
```

Comment:
Did we simulate such an error?

Reply:
Nope, but that's Biovalidator's error for when `maxItems` : 1 fails